src/components/CreateSplit/CreateSplitForm.tsx

```diff
@@ -56,7 +55,7 @@ const CreateSplitForm = ({
 
   const onSubmit = useCallback(
     async (data: ICreateSplitForm) => {
-      await switchChain({ chainId: baseSepolia.id })
+      await switchChainAsync({ chainId })
```

Comment:
optional: we could remove all this `prop drilling` and just reference the `CHAIN` from shared `const` lib